Project/Engine/src/sysFrame/Object/Camera/Camera.cpp

```
@@ -9,6 +9,10 @@ Camera::Camera() {
     uuid_ = System::CreateUuid();
 }
 
+Camera::Camera(const std::string& id) {
+    uuid_ = id;
```

Cppcheck (2.10-2)

[performance] Variable 'uuid_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)